frontend/wallet/src/render/views/footer/Footer.re

```diff
@@ -131,10 +131,10 @@ let make = () => {
                        ) => {
                          let variables =
                            SendPayment.make(
-                             ~from=PublicKey.toString(from),
-                             ~to_=PublicKey.toString(to_),
-                             ~amount,
-                             ~fee,
+                             ~from=Apollo.Encoders.publicKey(from),
+                             ~to_=Apollo.Encoders.publicKey(to_),
+                             ~amount=Js.Json.string(amount),
```

Comment:
why not use `Apollo.Encoders.int64` here?

Reply:
Apollo.Encoders.int64 converts from int64 to Js.Json.t, here amount is a string

Reply:
oh duh, my b. It's weird though in the `validate` function in SendButton, we convert the fee & amount to int64 and then return strings, while we convert the toStr into a PublicKey.t and return that - should be consistent here

Reply:
fair enough

Reply:
This is taking too long, I'm going to merge and hopefully rework this stuff later